src/App.js

```diff
@@ -1,12 +1,17 @@
 import React from "react";
 import styles from "./App.module.scss";
-import Header from "./components/Header/Header";
 import { Route, Switch } from "react-router-dom";
+import Header from "./components/Header/Header";
 import NewsPage from "./containers/News/News";
 import SearchNewsPage from "./pages/SearchNewsPage";
 import StockPage from "./pages/StockPage";
 import Watch from "./pages/WatchPage";
 import CovidPage from "./pages/CovidPage";
+import ClipNewsPage from "./pages/ClipNewsPage";
+import LoginPage from "./pages/LoginPage";
+import SignupPage from "./pages/SignUpPage";
+import PrivateRoute from "./components/PrivateRoute/PrivateRoute";
```

Comment:
PrivateRoute.js -> index.js

```suggestion
import PrivateRoute from "./components/PrivateRoute";
```